PowerShell.Common.props

```diff
@@ -0,0 +1,23 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Product>PowerShell Core</Product>
+    <Company>Microsoft Corporation</Company>
+    <Copyright>(c) Microsoft Corporation. All rights reserved.</Copyright>
+
+    <FileVersion>6.0.0.0</FileVersion>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+    <NeutralLanguage>en-US</NeutralLanguage>
+
+    <ProductVersion>6.0.0-beta.3</ProductVersion>
+    <InformationalVersion>6.0.0-beta.3</InformationalVersion>
```

Comment:
I think `<ProductVersion>`  and `InformationalVersion` are probably not needed in this PR. The `target` change can bring them back.